src/components/Adyen/Adyen.js

```diff
@@ -73,10 +74,10 @@ const Adyen = ({
       paymentMethods.find(item => item.type === 'googlepay')?.configuration;
 
     const configuration = {
-      locale: 'en-US',
-      translations: adyenTranslations,
+      locale: adyenConfiguration?.locale || 'en-US',
+      translations: adyenConfiguration?.translations || adyenTranslations,
```

Comment:
in our `adyenTranslations` we have a few translations. Maybe we could merge these two objects (our and from prop), where that one from the props has priority?